P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+              who="#WU00010808" xml:id="cmc_post13">
+              <p> I would kindly request from Mr. Meyer to allow others to edit the [...]</p>
+            </post>
+            <post indentLevel="1" modality="written" when-iso="2006-09-08T03:49+00"
+              who="#WU00010804" xml:id="cmc_post14">
+              <p>I dont agree, this article is not about Dr. Meyer, [...]</p>
+            </post>
+            <post indentLevel="2" modality="written" when-iso="2006-09-08T04:16+00"
+              who="#WU00005520" xml:id="cmc_post15">
+              <p>Why don't you read the policy. [...]</p>
+            </post>
+            <post indentLevel="3" modality="written" when-iso="2006-11-01T22:58+00"
+              who="#WU00010815" xml:id="cmc_post16">
+              <p>Because the policy makes no sense, [...]</p>
+            </post>
+          </div>
+        </egXML>
+      </p>
+    </div>
+    <div xml:id="CMCcmcatts">
+      <head>Attributes for General CMC Encoding</head>
+      <p>The attribute <att>generatedBy</att> is also unique to CMC encoding. But unlike
+          <att>modality</att>, <att>replyTo</att>, and <att>indentLevel</att>, it is available not
+        only on the <gi>post</gi> element, but on any of its descendants as well. <specList>
+          <specDesc key="att.cmc" atts="generatedBy"/>
+        </specList>
+      </p>
+      <p>The <att>generatedBy</att> attribute may indicate, for <gi>post</gi> or any of its
+        descendants, how the content transcribed in an element was generated in a CMC environment.
+        That is, whether the source text being transcribed was created by a human user, created by
+        the CMC system at the request of a human user (e.g., when the user activates a template that
+        generates the content, such as in a signature), outright generated by the CMC system (e.g. a
+        status message or a timestamp), or generated by an automated process external to the CMC
+        system itself. This attribute is optional; when it is not specified on a <gi>post</gi>
+        element its value is presumed to be <val>unspecified</val>; when it is unspecified on any
+        descendant of <gi>post</gi> its value is inherited from the immediately enclosing element.
+        (And, in turn, if <att>generatedBy</att> is not specified on that element it inherits the
+        value from its immediately enclosing element, and so on up the document hierarchy until a
+          <gi>post</gi> is reached; the <gi>post</gi> either has a <att>generatedBy</att> attribute
+        specified or its presumed value is <val>unspecified</val>).</p>
+      <!-- BEGIN duplicative section %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% … -->
+      <!--
+          If you make a change here, make it in the att.cmc.xml file,
+          as well!
+      -->
+      <p>A list of suggested values for <att>generatedBy</att> follows: <list type="gloss">
```

Comment:
Review the punctuation of this list. Missing closing parentheses in items 2 and 3, and the use of period at the end of the item seems inconsistent

Reply:
Addressed, at least to some extent, in https://github.com/TEIC/TEI/commit/759c2adbaa0df79dd9f48e30060e2cf304e17a68. Take a look and see if you think it is OK now.